src/explorer/assets/main.html

```diff
@@ -0,0 +1,22 @@
+<!-- Mimi UI -->
+
+<html>
+<header>
+    
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vega/3.0.8/vega.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vega-lite/2.0.3/vega-lite.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vega-embed/3.0.0-rc7/vega-embed.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+
+</header>
+
+<body>
+
+    <div id="variablelist" class="tab"></div>
+    <div id="vis" class="graphic"></div>
+
+    <script src="refresh.js"></script>
+
+</body>
+
+</html>
```

Comment:
Very minor nitpick: can you always include one line break at the end of each file?